packages/accordions/stories/2-Accordion.stories.tsx

```diff
@@ -0,0 +1,320 @@
+/**
```

Comment:
Is there a particular reason files under `/accordions/stories` are prefixed with a number like `1-filename.ext`? Should files under `/tabs/stories` follow a number prefix naming?

Reply:
This is to ensure the correct ordering of stories:
- `README` first
- Alphabetical groups after that

I've seen this method used in other storybooks, but if there is a cleaner way I would like to get rid of these.